Update tidb-lightning-overview.md (#7514)
pingcap/docs: tidb-lightning/tidb-lightning-overview.md

Currently, TiDB Lightning supports:

> - Importing files exported by [Dumpling](/dumpling-overview.md), CSV files, and [Apache Parquet files generated by Amazon Aurora](/migrate-from-aurora-using-lightning.md).

Reviewer: ```suggestion
- Importing files exported by [Dumpling](/dumpling-overview.md), CSV files, and [Apache Parquet files generated by Amazon Aurora](/migrate-aurora-to-tidb.md).
```

> - The data source of the [Dumpling](/dumpling-overview.md), CSV or standard parquet file such as [Amazon Aurora Parquet](/migrate-from-aurora-using-lightning.md) exported formats.

Reviewer: ```suggestion
- Importing files exported by [Dumpling](/dumpling-overview.md), CSV files, and [Apache Parquet files generated by Amazon Aurora](/migrate-from-aurora-using-lightning.md).
```

- If you use TiDB Lighting together with TiDB:

    TiDB Lightning does not support importing `charset=GBK` tables to TiDB clusters earlier than v5.4.0.

> - When using parquet file to import, lightning currently only accepts the data types supported in Amazon Aurora Parquet.

Reviewer: ```suggestion
- For Apache Parquet files, TiDB Lightning currently only accepts Amazon Aurora Parquet files.
```
